.github/workflows/cwltool.yml

```
@@ -23,7 +30,7 @@ jobs:
           python-version: '3.9.x'
       
       - name: Setup prerequirements
-        run: pip install cwltest cwltool
+        run: pip install cwltest cwltool udocker && udocker install
       
       - name: Run tests against the reference runner
-        run: ./run_test.sh RUNNER=cwltool EXTRA=--parallel -j$(nproc)
+        run: ./run_test.sh RUNNER=cwltool EXTRA="--parallel --user-space-docker-cmd=${{ matrix.docker }}" -j$(nproc)
```

```suggestion
        run: cwltest --junit-verbose --verbose --tool cwltool --test=conformance_tests.yaml -j$(nproc) -- "--parallel --user-space-docker-cmd=${{ matrix.docker }} --debug"
```

Direct running `cwltest` so we can pass `--junit-verbose` to get rid of `--quiet` which would otherwise prevent us from adding `--debug`